charts/backstage/README.md.gotmpl

```diff
@@ -38,7 +38,7 @@ This chart bootstraps a [Backstage](https://backstage.io/docs/deployment/docker)
 ## Prerequisites
 
 - Kubernetes 1.19+
```

Comment:
I would also suggest we move up to a newer k8s minimum version, to align with the current EUS version of Openshift 4.12. 

```suggestion
- Kubernetes 1.25+
```